src/components/TopBar.tsx

```diff
@@ -116,25 +139,32 @@ export const TopBar: React.SFC<Props> = (props) => (
   <Container>
     <Bar>
       <BarContainer>
-        <Logo src="/assets/offering/logo.png" />
+        <EscapeLink href="/hedvig">
```

Comment:
should this be a link to `https://hedvig.com` rather?

Reply:
I'm also leaning towards that, but I guess it depends on what makes most sense in the context of web onboarding